terraform/modules/aws/vpc/main.tf

```diff
@@ -0,0 +1,119 @@
+resource "aws_vpc" "main" {
+  cidr_block           = var.cidr_block
+  enable_dns_hostnames = var.enable_dns_hostnames
+
+  tags = {
+    Name = var.name
+  }
+}
+
+resource "aws_eip" "eip_nat" {
+  for_each = var.nat_gws
+  tags = {
+    Name = var.eip_name
```

Comment:
so every nat will have the same name :thinking:. Maybe we need to add some subnet info to it?

Reply:
ok. done